### DISCUSSION TITLE
Vim — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#240] [Announcements]
https://mazhuang.org/wiki/vim/

**Comment by QingyiZhang** [comment]
以下存在笔误，建议修正
将全文中的 str1 替换为 str1	:%s/str1/str2/g  ---->将全文中的 str1 替换为 str2
将 1 到 5 行中的 str1 替换为 str2	:1,5/str1/str2/g ------->:1,5s/str1/str2/g

**Comment by mzlogin** [comment]
> @QingyiZhang
> 以下存在笔误，建议修正
> 将全文中的 str1 替换为 str1	:%s/str1/str2/g  ---->将全文中的 str1 替换为 str2
> 将 1 到 5 行中的 str1 替换为 str2	:1,5/str1/str2/g ------->:1,5s/str1/str2/g

确实如此，已经修复提交了，感谢反馈！
